packages/casting/src/follower-cosmjs.js

```diff
@@ -136,7 +136,7 @@ export const makeCosmjsFollower = (
       const info = await client.abciInfo();
       const { lastBlockHeight } = info;
       assert.typeof(lastBlockHeight, 'number');
-      return lastBlockHeight;
+      return { blockHeight: lastBlockHeight, endpoint };
```

Comment:
a little awkward to be passing around the `endpoint`. that should be encapsulated in the leader, which I think we could do by refactoring to have a single callback to `mapEndpoints` that does the right thing.

but given this will be refactored again with the cosmjs change, I think this is okay. consider a `xxx` comment about the factoring.